docs-ui/components/code.tsx

```diff
@@ -51,14 +52,22 @@ const Code = ({children, className, label}: Props) => {
     const copiableContent = children.replace(/\/\*[\s\S]*?\*\/|\/\/.*/g, '');
 
     copy(copiableContent);
-
     setCopied(true);
 
-    setTimeout(() => {
+    copyTimeoutRef.current = window.setTimeout(() => {
       setCopied(false);
     }, 500);
   }
 
+  // Cleanup timeout on component unmount
+  useEffect(() => {
```

Comment:
Looks like this might have snuck in here as well, but not worried about it. Good cleanup